common/djangoapps/student/views/dashboard.py

```diff
@@ -519,6 +520,7 @@ def get_dashboard_course_limit():
     return course_limit
 
 
+@view_namespace('students.views.dashboard')
```

Comment:
This is just a sample usage, not a suggestion or recommendation for how the namespaces should be named.